example/android/app/build.gradle

```diff
@@ -25,8 +24,8 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.mythichelm.localnotificationsexample"
-        minSdkVersion 16
-        targetSdkVersion 25
+        minSdkVersion 23
```

Comment:
same feedback here around minSdkVersion. Ideally we want to support the minimum that Flutter supports (currently 16)

Reply:
I think this is only for the example app. I can probably set this down to 16, but as I said in https://github.com/mitchhymel/local_notifications/pull/6#discussion_r178462340 , I would rather stick to 23, till be refactor the android code